pkg/bootstrap/config.go

```diff
@@ -302,9 +302,14 @@ func getProxyConfigOptions(config *meshAPI.ProxyConfig, metadata *model.Bootstra
 			opts = append(opts, option.DataDogAddress(tracer.Datadog.Address))
 		case *meshAPI.Tracing_Stackdriver_:
 			var projectID string
-			var err error
-			if projectID, err = md.ProjectID(); err != nil {
-				return nil, fmt.Errorf("unable to process Stackdriver tracer: %v", err)
+			var projFound bool
+
+			if projectID, projFound = metadata.PlatformMetadata[platform.GCPProject]; !projFound {
```

Comment:
Is `platform.GCPProject` a dup of `platform.GCPProjectNumber`? IIRC `GCPProject` is the id, while `GCPProjectNumber` is the numeric id. Should we make the logic consistent and only try fetching the `GCPProject`.

Reply:
They aren't dupes, but they represent the same info in two different ways. I can't recall the context here at the moment, but I think maybe some of the tests only set the project number somehow. Worth investigating perhaps.

Reply:
I think the tracer client expect this `projectID` to be the ID instead of numeric ID. So should we remove this fallback?

Reply:
Removed.